packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -1,11 +1,13 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+/// @title Module for electing a council, represented by a set of NFT holders
```

Comment:
Thought it'd be more helpful to front end developers and consumers of our contracts to move the docs to the interface.